### ISSUE TITLE
Search and categories filtering suggestion

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/themedd#43]
As per the EDD site itself, it would be useful to have the ability to filter the products by categories and also search on the download archive pages.

These settings could be optional in EDD settings within the theme.

**Comment by amdrew** [comment]
👍 

Related: https://github.com/easydigitaldownloads/themedd/issues/30

**Comment by amdrew** [comment]
@davidsherlock For this, are you referring to the grey buttons here next to categories?

<img width="1173" alt="screenshot 2017-09-20 17 29 14" src="https://user-images.githubusercontent.com/52581/30628246-3ceaa294-9e29-11e7-90b3-323e6839060b.png">

**Comment by rubengc** [comment]
This can be done with a free plugin, and results will be loaded using ajax:
https://wordpress.org/plugins/edd-ajax-filters/

Also includes options to animate results (in and out animations)

There is a page using it and with the same approach as EDD official site:
https://gamipress.com/add-ons/

**Comment by davidsherlock** [comment]
@amdrew, yes I am. It would be useful to have a large search box above the download lists with search and categories displayed in a similar fashion to the EDD site.
